pom.xml

```diff
@@ -34,12 +34,10 @@
     <description>Runs JUnit tests in an OSGi framework and provides the JUnit libraries</description>
     
     <properties>
-        <junit.version>4.13</junit.version>
+        <junit.version>4.13.2</junit.version>
         <hamcrest.version>1.3</hamcrest.version>
         <jacoco.version>0.6.2.201302030002</jacoco.version>
-        <junit-jupiter.version>5.6.0</junit-jupiter.version>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <junit-jupiter.version>5.10.0</junit-jupiter.version>
```

Comment:
We could bump it to 5.10.1 already.

Reply:
After bumping, following the encounter with an issue, I've identified a test failure. I'll proceed to generate a separate ticket to address this issue. Therefore, I'll maintain the current version at 5.10.0 for the time being.

```
[ERROR] Failures:
[ERROR]   OSGiAnnotationTest.injectServices:110->withMockedFrameworkUtil:352->lambda$injectServices$1:111->assertNoFailures:382 number of tests found
Expected: a value greater than <0>
     but: <0> was equal to <0>
[INFO]
```